packages/desktop-gui/src/lib/browser-model.js

```diff
@@ -25,6 +26,14 @@ export default class Browser {
   }
 
   @computed get icon () {
-    return 'chrome'
+    if (this.family === 'electron') {
+      return 'chrome'
```

Comment:
it would be nicer to get separate electron icon I think

Reply:
yeah, using their icon would cut down on people confusing electron w/ chrome too: 
![image](https://user-images.githubusercontent.com/1151760/69734815-a5a44d00-10fd-11ea-8422-c45ef6c74578.png)

Reply:
we're using font-awesome 4.7 here so 😬😬😬